src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size;
+    private int currentCapacity;
```

Comment:
what do you think, can we remove this field and use `values.length` when we need this value?

Reply:
done